src/sql/SelectStatement.h

```diff
@@ -58,7 +75,19 @@ namespace hsql {
     Expr* whereClause;
     GroupByDescription* groupBy;
 
-    SelectStatement* unionSelect;
+    // Note that a SetOperator is always connected to a
+    // different SelectStatement. This statement can itself
+    // have SetOperator connections to other SelectStatements.
+    // To evaluate the operators in the correct order: 
+    //    1. Fully evaluate the nested SelectStatement
+    //    2. Connect the nested SelectStatement to our original
+    //       with the given SetOperation
+    //    3. Connect the result with the next SetOperator in the
+    //       vector
+    //    4. Repeat until all SetOperators in the vector are
+    //       connected 
+    std::vector<SetOperator*>* setOperators;
```

Comment:
How do you feel about „setDefinition(s)“?

Reply:
Only for the variable name or also the class names?

Reply:
I don't see a reason for the two not to be equal.

Reply:
Then I'd maybe prefer "setOperatorDefinitions" it's getting a bit verbose but seems to get closer to what we are trying to say

Reply:
I guess I don't like the "operator" part of it. `setOperation` would be slightly better, too.

*There are 2 hard problems in computer science: cache invalidation, naming things, and off-by-1 errors.*

Reply:
Well, this is not helped by the fact that setAnything basically sounds like a setter for stuff. That's why I like the operator behind it to make it clearer

Reply:
I'd say `setOperation` is best then